app/data/models/delinkedcalculation.js

```diff
@@ -0,0 +1,45 @@
+module.exports = (sequelize, DataTypes) => {
+  const number0 = 0
+  const number11 = 11
```

Comment:
is there a way we can name number0 and number11 more informatively? so that they mean something to people rather than a generic name